### DISCUSSION TITLE
240322

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#69] [팀회의]

**Comment by jisunipark** [comment]
## 논의사항
- 오늘 회의 시간에 컬러 팔레트 설정하는 것으로 이야기 되어서 어제 멘토링 때 질문 드렸던 동적 스타일링 문제는 해결되었습니다.

## 어제 한 일
- 자유게시판 / 이슈 페이지 ui
- 그룹페이지 탭 기능

## 오늘 할 일
- 그룹 페이지 탭 기능
- 사이드바, 이슈 페이지 api

## 오늘의 일정 (코어 타임)
- x

## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
크게 없는 것 같아요.. 죄송합니다 오늘은 쭉 할 수 있습니다 !
## 오늘 할 일
메인페이지 캘린더 부분 데이터 연동, 나의 페이지 캘린더 데이터 연동 및 코드 리팩토링
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
토큰 갱신, 회원가입 페이지 정리하다가 말음
## 오늘 할 일
리팩토링 끝내기
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
